# HG changeset patch
# User Rik <rik@octave.org>
# Date 1375939232 25200
#      Wed Aug 07 22:20:32 2013 -0700
# Node ID d87179b38bcf19b5621e70b829dea558fc6c2392
# Parent  efd8963f925f07cc91b20c7054f1b235efe5b2a4
doc: Touch up alignment of @multitables in plot, legend, newplot.

* scripts/plot/legend.m, scripts/plot/newplot.m,
scripts/plot/plot.m: Don't use spacer rows between multitable items.

diff --git a/scripts/plot/legend.m b/scripts/plot/legend.m
--- a/scripts/plot/legend.m
+++ b/scripts/plot/legend.m
@@ -38,48 +38,27 @@
 ## @var{hobjs}, are not specified then the legend's strings will be associated
 ## with the axes' descendants.  @code{legend} works on line graphs,
 ## bar graphs, etc.  A plot must exist before legend is called.
 ##
 ## The optional parameter @var{pos} specifies the location of the legend
 ## as follows:
 ##
 ## @multitable @columnfractions 0.06 0.14 0.80
-##
-## @headitem @tab @var{pos} @tab
-##   location of the legend
-##
-## @item @tab north @tab
-##   center top
-##
-## @item @tab south @tab
-##   center bottom
-##
-## @item @tab east @tab
-##   right center
-##
-## @item @tab west @tab
-##   left center
-##
-## @item @tab northeast @tab
-##   right top (default)
-##
-## @item @tab northwest @tab
-##   left top
-##
-## @item @tab southeast @tab
-##   right bottom
-##
-## @item @tab southwest @tab
-##   left bottom
-##
+## @headitem @tab pos @tab location of the legend
+## @item @tab north @tab center top
+## @item @tab south @tab center bottom
+## @item @tab east @tab right center
+## @item @tab west @tab left center
+## @item @tab northeast @tab right top (default)
+## @item @tab northwest @tab left top
+## @item @tab southeast @tab right bottom
+## @item @tab southwest @tab left bottom
 ## @item
-##
-## @item @tab outside @tab
-##   can be appended to any location string
+## @item @tab outside @tab can be appended to any location string
 ## @end multitable
 ##
 ## The optional parameter @var{orient} determines if the key elements
 ## are placed vertically or horizontally.  The allowed values are
 ## "vertical" (default) or "horizontal".
 ##
 ## The following customizations are available using @var{option}:
 ##
diff --git a/scripts/plot/newplot.m b/scripts/plot/newplot.m
--- a/scripts/plot/newplot.m
+++ b/scripts/plot/newplot.m
@@ -38,34 +38,32 @@
 ## set to "on".  Set NextPlot property to "add".  This typically clears a
 ## figure, but leaves in place hidden objects such as menubars.  This is
 ## equivalent to @code{clf}.
 ##
 ## @item "replace" @tab Delete all child objects of the figure and reset all
 ## figure properties to their defaults.  However, the following four properties
 ## are not reset: Position, Units, PaperPosition, PaperUnits.  This is
 ## equivalent to @code{clf reset}.
-##
 ## @end multitable    
 ##
 ## @multitable @columnfractions .25 .75
 ## @headitem Axis NextPlot @tab Action
 ## @item "add" @tab Add new graphic objects to the current axes.  This is
 ## equivalent to @code{hold on}.
 ##
 ## @item "replacechildren" @tab Delete child objects whose HandleVisibility is
 ## set to "on", but leave axis properties unmodified.  This typically clears a
 ## plot, but preserves special settings such as log scaling for axes.
 ## This is equivalent to @code{cla}.
 ##
 ## @item "replace" (default) @tab Delete all child objects of the axis and
 ## reset all axis properties to their defaults.  However, the following
 ## properties are not reset: Position, Units.  This is equivalent to
 ## @code{cla reset}.
-##
 ## @end multitable    
 ##
 ## If the optional input @var{hfig} or @var{hax} is given then prepare the
 ## specified figure or axes rather than the current figure and axes.
 ##
 ## The optional return value @var{hax} is a graphics handle to the created
 ## axes object (not figure).
 ##
diff --git a/scripts/plot/plot.m b/scripts/plot/plot.m
--- a/scripts/plot/plot.m
+++ b/scripts/plot/plot.m
@@ -105,110 +105,89 @@
 ## given, then the default plot style is solid lines with no markers and the
 ## color determined by the "colororder" property of the current axes.
 ##
 ## Format arguments:
 ##
 ## @table @asis
 ## @item linestyle
 ##
-## @table @asis
-## @item @samp{-}   Use solid lines (default).
-##
-## @item @samp{--}  Use dashed lines.
-##
-## @item @samp{:}   Use dotted lines.
-##
-## @item @samp{-.}  Use dash-dotted lines.
-## @end table
+## @multitable @columnfractions 0.06 0.94
+## @item @samp{-}  @tab Use solid lines (default).
+## @item @samp{--} @tab Use dashed lines.
+## @item @samp{:}  @tab Use dotted lines.
+## @item @samp{-.} @tab Use dash-dotted lines.
+## @end multitable
 ##
 ## @item markerstyle
 ##
-## @table @asis
-## @item @samp{+}  crosshair
-##
-## @item @samp{o}  circle
-##
-## @item @samp{*}  star
-##
-## @item @samp{.}  point
-##
-## @item @samp{x}  cross
-##
-## @item @samp{s}  square
-##
-## @item @samp{d}  diamond
-##
-## @item @samp{^}  upward-facing triangle
-##
-## @item @samp{v}  downward-facing triangle
-##
-## @item @samp{>}  right-facing triangle
-##
-## @item @samp{<}  left-facing triangle
-##
-## @item @samp{p}  pentagram
-##
-## @item @samp{h}  hexagram
-## @end table
+## @multitable @columnfractions 0.06 0.94
+## @item @samp{+} @tab crosshair
+## @item @samp{o} @tab circle
+## @item @samp{*} @tab star
+## @item @samp{.} @tab point
+## @item @samp{x} @tab cross
+## @item @samp{s} @tab square
+## @item @samp{d} @tab diamond
+## @item @samp{^} @tab upward-facing triangle
+## @item @samp{v} @tab downward-facing triangle
+## @item @samp{>} @tab right-facing triangle
+## @item @samp{<} @tab left-facing triangle
+## @item @samp{p} @tab pentagram
+## @item @samp{h} @tab hexagram
+## @end multitable
 ##
 ## @item color
 ##
-## @table @asis
-## @item @samp{k}  blacK
-##
-## @item @samp{r}  Red
-##
-## @item @samp{g}  Green
-##
-## @item @samp{b}  Blue
-##
-## @item @samp{m}  Magenta
-##
-## @item @samp{c}  Cyan
-##
-## @item @samp{w}  White
-## @end table
+## @multitable @columnfractions 0.06 0.94
+## @item @samp{k} @tab blacK
+## @item @samp{r} @tab Red
+## @item @samp{g} @tab Green
+## @item @samp{b} @tab Blue
+## @item @samp{m} @tab Magenta
+## @item @samp{c} @tab Cyan
+## @item @samp{w} @tab White
+## @end multitable
 ##
 ## @item ";key;"
 ## Here "key" is the label to use for the plot legend.
 ## @end table
 ##
 ## The @var{fmt} argument may also be used to assign legend keys.
 ## To do so, include the desired label between semicolons after the
-## formatting sequence described above, e.g., "+b;Key Title;"
-## Note that the last semicolon is required and Octave will generate an error
-## if it is left out.
+## formatting sequence described above, e.g., "+b;Key Title;".
+## Note that the last semicolon is required and Octave will generate
+## an error if it is left out.
 ##
 ## Here are some plot examples:
 ##
 ## @example
 ## plot (x, y, "or", x, y2, x, y3, "m", x, y4, "+")
 ## @end example
 ##
 ## This command will plot @code{y} with red circles, @code{y2} with solid
 ## lines, @code{y3} with solid magenta lines, and @code{y4} with points
 ## displayed as @samp{+}.
 ##
 ## @example
 ## plot (b, "*", "markersize", 10)
 ## @end example
 ##
 ## This command will plot the data in the variable @code{b},
-## with points displayed as @samp{*} with a marker size of 10
+## with points displayed as @samp{*} and a marker size of 10.
 ##
 ## @example
 ## @group
 ## t = 0:0.1:6.3;
 ## plot (t, cos(t), "-;cos(t);", t, sin(t), "-b;sin(t);");
 ## @end group
 ## @end example
 ##
 ## This will plot the cosine and sine functions and label them accordingly
-## in the key.
+## in the legend.
 ##
 ## If the first argument @var{hax} is an axes handle, then plot into this axis,
 ## rather than the current axes returned by @code{gca}.
 ##
 ## The optional return value @var{h} is a vector of graphics handles to
 ## the created line objects.
 ##
 ## To save a plot, in one of several image formats such as PostScript
